universe/UniverseObject.h

```diff
@@ -29,6 +29,10 @@ FO_COMMON_API extern const int INVALID_OBJECT_ID;
 // The ID number assigned to temporary universe objects
 FO_COMMON_API extern const int TEMPORARY_OBJECT_ID;
 
+// The ID number assigned instead of the system which represents the whole imperium
+// Used for industrial stockpile
+FO_COMMON_API extern const int IMPERIAL_OBJECT_ID;
```

Comment:
Understood, thank you for the good explanations. As a sidenote: I'd prefer that all my IDs would share the same pool. Usually always corner cases emerge where it helps to be able to handle different things in the same way. If we would have kept the stockpile locations one could distinguish your own empire (imperial stock) from one of your planets (local stock) by the id.